SimPEG/Optimization.py

```diff
@@ -939,6 +943,72 @@ def approxHinv(self, value):
         self._approxHinv = value
 
     @Utils.timeIt
+    def minimize(self, evalFunction, x0):
```

Comment:
the ones in test/base/optimize.py are closest to what you might want to include here: https://github.com/simpeg/simpeg/blob/master/tests/base/test_optimizers.py